src/common/constants.js

```diff
@@ -0,0 +1,24 @@
+const TEST_EMAIL = 'test@codeit.com';
+const TEST_PASSWORD = 'codeit101';
+
+const EMAIL_CHECK_MESSAGE = '이메일을 확인해 주세요.';
+const PASSWORD_CHECK_MESSAGE = '비밀번호를 확인해 주세요.';
+const EMPTY_EMAIL_MESSAGE = '이메일을 입력해 주세요.';
+const EMPTY_PASSWORD_MESSAGE = '비밀번호를 입력해 주세요.';
+const ERROR_EMAIL_MESSAGE = '올바른 이메일 주소가 아닙니다.';
+const DUPLICATE_EMAIL_MESSAGE = '이미 사용 중인 이메일입니다.';
+const ERROR_INVALID_PASSWORD_MESSAGE = '비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.';
+const ERROR_MATCH_PASSWORD_MESSAGE = '비밀번호가 일치하지 않아요.';
+
+export {
+  TEST_EMAIL,
+  TEST_PASSWORD,
+  EMAIL_CHECK_MESSAGE,
+  PASSWORD_CHECK_MESSAGE,
+  EMPTY_EMAIL_MESSAGE,
+  EMPTY_PASSWORD_MESSAGE,
+  ERROR_EMAIL_MESSAGE,
+  DUPLICATE_EMAIL_MESSAGE,
+  ERROR_INVALID_PASSWORD_MESSAGE,
+  ERROR_MATCH_PASSWORD_MESSAGE,
+};
```

Comment:
@JeongSabibi 

[제안]

```suggestion

export const EMAIL_ERROR_MESSAGE= {
CHECK: '이메일을 확인해 주세요.',
....
}

export const PASSWORD_ERROR_MESSAGE ={
CHECK: '비밀번호를 확인해 주세요.',
...
}

```